src/cascadia/WindowsTerminal/WindowEmperor.cpp

```diff
@@ -105,7 +106,9 @@ bool WindowEmperor::HandleCommandlineArgs()
     GetStartupInfoW(&si);
     const uint32_t showWindow = WI_IsFlagSet(si.dwFlags, STARTF_USESHOWWINDOW) ? si.wShowWindow : SW_SHOW;
 
-    Remoting::CommandlineArgs eventArgs{ { args }, { cwd }, showWindow };
+    const auto currentEnv{ til::env::from_current_environment() };

```

Comment:
Yep, it's **this easy** to package up the calling `wt`'s env vars into a string that we can move around easily.

Reply:
this takes care of the embedded null bytes? wow